app/services/PsrSubmissionService.scala

```diff
@@ -55,13 +56,19 @@ class PsrSubmissionService @Inject()(
 
   def submitPsrDetailsWithUA(
     srn: Srn,
-    userAnswers: UserAnswers
+    userAnswers: UserAnswers,
+    optFallbackCall: Option[Call] = None
```

Comment:
Why are we using this fallbackCall as an option ? Couldn't see anywhere sent None.

Reply:
As this was introduced gradually across all the 28 commits :) 
We can refactor later if you think that is useful.